test/util/test_runners/params.py

```diff
@@ -0,0 +1,29 @@
+import os
+from test.util.test_path_provider import TestRunnerPathProvider, TestIntegrationPathProvider
+
+
+class ClientParams:
+    """
+    DTO wrapped the client params
+    Used for integration tests
+    """
+
+    def __init__(self, hostname='https://conjur-https', username='admin', account='dev', api_key=None):
+        self.hostname = hostname
+        self.username = username
+        self.account = account
+        self.env_api_key = api_key if api_key else os.environ[
+            'CONJUR_AUTHN_API_KEY']  # api_key is None when we run the tests using nose2 script
```

Comment:
regarding your comment:
> # api_key is None when we run the tests using nose2 script

What do you mean here?

Reply:
api_key = None that we get in the constructor